bundles/CoreBundle/Resources/migrations/Version20190408084129.php

```diff
@@ -14,6 +14,7 @@ public function up(Schema $schema)
     {
         $this->addSql("ALTER TABLE `quantityvalue_units`
             CHANGE `baseunit` `baseunit` INT(11) UNSIGNED,
+            ADD `converter` VARCHAR(255),
```

Comment:
please don't forget to add this to install.sql

Reply:
Thanks for your hint. Added database changes to install.sql